### PR TITLE
S588850: Linked coupling pyaedt support

### PR DESCRIPTION
**Pull request by jianwei425**

### CODE REVIEW

File: examples/03-Circuit/EMIT_HFSS_Example.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+temp_folder = os.path.join(tmpfold, ("EmitHFSSExample"))
+if not os.path.exists(temp_folder):
+    os.mkdir(temp_folder)
+
+example_name = "Cell Phone RFI Desense"
+example_aedt = example_name + ".aedt"
+example_lock = example_aedt + ".lock"
+
+example_project = os.path.join(local_path, "example_models", example_aedt)
```
Comment (maxcapodi78) [inline examples/03-Circuit/EMIT_HFSS_Example.py:51]: @jianwei425 the examples files of published examples have not to be in that folder. On monday I can explain where to put it and to download from Examples.

File: pyaedt/emit_core/Couplings.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
+"""
```
Comment (maxcapodi78) [inline pyaedt/emit_core/Couplings.py:1]: I would suggest to not create another folder. I think this file could stay in modules or modeler folders.

**Review by maxcapodi78 — CHANGES_REQUESTED** [review]
overall looks good. A couple of modifications are needed. Especially the example

File: pyaedt/emit_core/Couplings.py (hunk 2 above)
Comment (jianwei425) [reply pyaedt/emit_core/Couplings.py:1]: @maxcapodi78 @myoung301, we thought we would group all EMIT only files under this emit_core foulder, this Coupling.py is the first one, there will be files for result nodes, and more

File: examples/03-Circuit/EMIT_HFSS_Example.py (hunk 1 above)
Comment (jianwei425) [reply examples/03-Circuit/EMIT_HFSS_Example.py]: decided to copy it from the EDT install and not saving a copy of the example file in pyaedt

**Pull request merged**